src/models/constructor-items/sub-blocks.ts

```diff
@@ -134,6 +135,18 @@ export interface BannerCardProps {
 
 export interface MediaCardProps extends MediaProps, AnalyticsEventsBase, CardBaseProps {}
 
+export interface PriceCardProps extends CardBaseProps, Pick<ContentBlockProps, 'theme'> {
+    title: string;
+    price: string;
+    pricePeriod?: string;
+    priceDetails?: string;
+    description?: string;
+    button?: ButtonProps;
+    link?: LinkProps;
+    backgroundColor?: string;
+    list?: string[];
```

Comment:
what kind of list ? maybe we should give a more semantic name, for example `priceList` ?